sdktests/server_side_stream_retry.go

```diff
@@ -122,7 +122,7 @@ func doServerSideStreamRetryTests(t *ldtest.T) {
 		expectNoMoreRequests(t, streamEndpoint)
 	}
 
-	t.Run("retry after I/O error on initial connect", func(t *ldtest.T) {
+	t.Run("retry after IO error on initial connect", func(t *ldtest.T) {
```

Comment:
I don't understand why you felt the need to copyedit this. I/O is a perfectly normal way to write that term in English. The fact that it's written `IO` in various function/type names is irrelevant.

Reply:
Sorry, shouldn't have added this with no explanatory comment. This isn't arbitrary copy-editing - the framework is unable to suppress these tests since it parses the `/` as part of the test path separator. 

The framework uses the `(t TestID) Plus` method to build up the test ID. It will append `streaming/retry behavior` to `retry after I/O error on reconnect`, the name of the test, resulting in segments:
```
streaming
retry behavior
retry after I/O error on reconnect
```

But when parsing `-skip` arguments, the framework uses ` ParseTestIDPattern` which uses:
```
parts := strings.Split(s, "/")
```
resulting in:
```
streaming
retry behavior
retry after I
O error on reconnect
```
which will not match.

An alternative to all of this would be treating the suppression test IDs as literal strings, and store them in a different variable altogether instead of using the `skip` API. Then we'd need to figure out how `skip/run` interacts with the suppressions, which is why I originally used the `skip` method.

Reply:
Oh! Ha! Yeah, sorry, a slash in a test name is problematic. In fact it might even be worth enforcing this in the framework - like, have it actually fail the test as soon as `t.Run` is called with a name like that, since it's extremely likely to cause problems.

Reply:
But I can fix that separately. This change is fine. The only one of my comments here that I care about is the misleading "test service error" message.